src/sinks/elasticsearch.rs

```diff
@@ -75,14 +84,19 @@ fn es(config: ElasticSearchConfig, acker: Acker) -> super::RouterSink {
     } else {
         Template::from("vector-%Y.%m.%d")
     };
-    let doc_type = config.clone().doc_type.unwrap_or("_doc".into());
+    let doc_type = config.doc_type.clone().unwrap_or("_doc".into());
 
     let policy = FixedRetryPolicy::new(
         retry_attempts,
         Duration::from_secs(retry_backoff_secs),
         HttpRetryLogic,
     );
 
+    let authorization = config.basic_auth.clone().map(|auth| {
+        let token = format!("{}:{}", auth.user, auth.password);
+        format!("Basic {}", base64::encode(token.as_bytes()))
```

Comment:
I think this is correct, but wanted to make sure [`base64url`](https://docs.rs/data-encoding/1.2.0/data_encoding/base64url/index.html) is not required here instead.

Reply:
I don't believe so, since it's not going into the url.

Reply:
AFAIK the authorization token uses the standard base64 character set.